styles.css

```diff
@@ -0,0 +1,54 @@
+#title {
+    text-align: center;
+}
+
+#survey-form {
+    text-align: center;
+    width: 40%;
+    background-color: grey;
```

Comment:
- [ ] Kindly, make sure that Your suggested changes phrased the comments politely, as suggestions instead of orders and always includes an explanation, a "why", for each requested change and comments are clear and easy to understand.

Reply:
changged it to light blue

Reply:
alright nice work

Reply:
alright i ll try to alter my comments but if I couldn't I ll ask for more change and will be polite in my comments

Reply:
- [ ] [REQUIRED ] Please make sure that you implemented changes requested by the previous reviewer before submitting and please note that it is important to be polite and always tell the reason, **WHY** you suggest something or request a change as requested by the previous reviewer :smile: 

> * [ ]  Good Job for using GitHub inline comments feature but the suggested changes doesn't have an explanation, a "why", for each requested change.
>   For example : "Kindly, I recommend you to adjust margin to 5px for label element , that would improve  the page more accessibility"

Reply:
- [ ] [REQUIRED ] Please make sure that you implemented changes requested by the previous reviewer before submitting and please note that it is important to be polite and always tell the reason, **WHY** you suggest something or request a change as requested by the previous reviewer :smile: 

> * [ ]  Good Job for using GitHub inline comments feature but the suggested changes doesn't have an explanation, a "why", for each requested change.
>   For example : "Kindly, I recommend you to adjust margin to 5px for label element , that would improve  the page more accessibility"

Reply:
- Hello @shobeee, I see that you have tried to implement some changes suggested by the previous reviewers, but there are still some issues with your review. As the previous reviewers have stated, it is important to state the "WHY" for suggesting changes, in this context a reason could be "...to improve the user interface and user experience".Please try to fix this 👍 👍

Reply:
I think its fixed as this change is for page accessibility I have mentioned it now.
please review it